common/scripted_triggers/wc_coa_triggers.txt

```diff
@@ -58,10 +66,19 @@ african_pagan_coa_trigger = {
 	coa_african_religion_trigger = yes
 }
 feudal_europe_coa_trigger = {
-	# Warcraft
-	like_christianity_religion_trigger = yes
+	OR = {
+		AND = {
+		# Warcraft
+		like_christianity_religion_trigger = yes
 
-	scope:culture = { has_graphical_culture = western_coa_gfx }
+		scope:culture = { has_graphical_culture = western_coa_gfx }
+		}
+		
+		#Warcraft
+		scope:culture = { 
+			has_graphical_culture = wc_pagan_coa_gfx
```

Comment:
`wc_pagan_coa_gfx` acts as a catch-all template for some more tribal cultures such as gnolls, kobolds, tauren, troggs and tuskarr. It enlists the same template used by `european_pagan_coa_trigger` which features abstract circles and symbols found here: (coa_templates.txt)

		# European Paganism - Single, Circle, Senester
		special_selection = {
			trigger = {
				# Warcraft
				european_pagan_coa_trigger = yes		
			}
			# Weights are temps
			16000 = basic_pagan_template
			16000 = basic_pagan_template_inverted			
			8000 = basic_pagan_template_senester
			8000 = basic_pagan_template_senester_inverted
			3000 = geometrical_pagan_template
			4000 = geometrical_pagan_template_inverted				
			3000 = geometrical_pagan_template_senester	
			2000 = circled_pagan_template
			2000 = circled_pagan_template_inverted
			2000 = circled_pagan_template_offset	
			2000 = circled_pagan_template_offset_inverted
		}

`wc_pagan_coa_gfx` is meant to provide more unique identity in cultural CoAs through the player experience since it does not feature any animals and is only used by minor cultures (excluding Tauren who also use `druidic_coa_gfx`). The other reason I decided against including animals for `wc_pagan_coa_gfx` is because it is a catch-all pagan template used for varying cultures, therefore it should be neutral. This brings me to my last point: animalistic pagan coas. From what I can tell, CK3 does not have any baseline animalistic cultural_coa_gfx; they are all refined by special selections such as sami, germanic, magyar, etc. 

One solution is adding a trigger to `pagan_single_charge_list` conditional on `has_graphical_culture = wc_pagan_coa_gfx`. This introduces animalistic coas to culturals using `wc_pagan_coa_gfx`. (I wanted to keep druidic_coa_gfx limited to druidic cultures only).

Reply:
> wc_pagan_coa_gfx acts as a catch-all template for some more tribal cultures such as gnolls, kobolds, tauren, troggs and tuskarr.

But `feudal_europe_coa_trigger` unlocks Feudal europe flags. Lions, crosses, etc.

Reply:
And locks pagan flags.
![image](https://user-images.githubusercontent.com/46384992/161389610-7b27ceb8-ce00-45d6-a923-47cc978e32df.png)

Reply:
You're right! I don't know why `wc_pagan_coa_gfx` was even put in `feudal_europe_coa_trigger` in the first place 😅